modules/pam_sepermit/tst-pam_sepermit-retval.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Check pam_sepermit return values.
+ */
+
+#include "test_assert.h"
+
+#include <limits.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <security/pam_appl.h>
+
+#define MODULE_NAME "pam_sepermit"
+#define TEST_NAME "tst-" MODULE_NAME "-retval"
+
+static const char service_file[] = TEST_NAME ".service";
+static struct pam_conv conv;
+
+int
+main(void)
+{
+  pam_handle_t *pamh = NULL;
+  FILE *fp;
+  char cwd[PATH_MAX];
+
+  ASSERT_NE(NULL, getcwd(cwd, sizeof(cwd)));
+
+  /* conf is an explicit file */
+  ASSERT_NE(NULL, fp = fopen(service_file, "w"));
+  ASSERT_LT(0, fprintf(fp, "#%%PAM-1.0\n"
+		       "auth     [success=done ignore=ignore default=bad] "
+		       " %s/.libs/%s.so conf=%s/tests/sepermit.conf\n",
+		       cwd, MODULE_NAME, cwd));
```

Comment:
This code assumes that `tests/sepermit.conf` is located in $(builddir), but in fact it's located in $(srcdir).
One of possible solutions is passing a macro with a correct path to `tests/sepermit.conf` from Makefile.am to this compilation unit.